devel/floppy/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       compilers 1.0
+
+name            floppy
+version         7.0
+revision        0
+categories      devel
+platforms       darwin
+license         Noncommercial
```

Comment:
Where did you see that these programs may only be used for noncommercial purposes?

What I see is that the file `copyright` in both programs says:

> This program may not be copied or otherwise distributed without permission.

So that's troublesome. To comply with that, we would have to list the license as `Restrictive NoMirror` to prevent us from distributing binaries and source code, respectively.

Reply:
OK. Done as suggested.